integration-test/amp-protection.spec.js

```diff
@@ -59,6 +59,12 @@ test.describe('Test AMP link protection', () => {
             await page.goto(
                 initialUrl, { waitUntil: 'commit' }
             )
+
+            if (page.url().startsWith('https://www.wpxi.com/unavailable-location/')) {
```

Comment:
Could you ping @SlayterDev to just have this link removed/changed in the privacy-test-pages project?